django_mongodb/fields/object_id.py

```diff
@@ -0,0 +1,30 @@
+from bson import ObjectId, errors
```

Comment:
is this file a required part of the change?

Reply:
This new field was used in the django library when a new model is created, not here. But we will handle those fields as string for now, so this field is not longer needed.